docs/extensions/get-started/generator.md

```diff
@@ -0,0 +1,65 @@
+# New Extension Project with Generator
+
+The [Lens Extension Generator](https://github.com/lensapp/generator-lens-ext) scaffolds a project ready for development. Install Yeoman and Lens Extension Generator with:
+
+```bash
+npm install -g yo generator-lens-ext
+```
+
+Run the generator and fill out a few fields for a TypeScript project:
+
+```bash
+yo lens-ext
+# ? What type of extension do you want to create? New Extension (TypeScript)
+# ? What's the name of your extension? my-first-lens-ext
+# ? What's the description of your extension? My hello world extension
+# ? What's your extension's publisher name? @my-org/my-first-lens-ext
+# ? Initialize a git repository? Yes
+# ? Install dependencies after initialization? Yes
+# ? Which package manager to use? yarn
+# ? symlink created extension folder to ~/.k8slens/extensions (mac/linux) or :User
+s<user>.k8slensextensions (win)? Yes
```

Comment:
This is more related to generator itself, but is the window path correct?

Reply:
Probably not... I copied from here https://docs.k8slens.dev/master/extensions/get-started/your-first-extension/#windows

And @nevalla do you know what's the same as ~/ on Windows? Now I remember I haven't implement symlink on Windoes yet https://github.com/lensapp/generator-lens-ext/issues/10 

Found this
https://stackoverflow.com/questions/9228950/what-is-the-alternative-for-users-home-directory-on-windows-command-prompt

Reply:
okay probably it's just missing backslash. Correct should be `<user>\.k8slens\extensions` also on windows.

Reply:
Yeah... not sure what happens to my copy-paste, let's blame Big Sur. fixed in https://github.com/lensapp/lens/pull/1411/commits/1d763a1877937e0616179e936ccf9e1aa6a629a7